src/Template/PhpTemplate/FilePathTemplateFactory.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+
+namespace Dhii\Output\Template\PhpTemplate;
+
+use Dhii\Output\PhpEvaluator\FilePhpEvaluatorFactoryInterface;
+use Dhii\Output\Template\PathTemplateFactoryInterface;
+use Dhii\Output\Template\PhpTemplate;
+use Dhii\Output\Template\TemplateInterface;
+
+/**
+ * @inheritDoc
```

Comment:
Missing `@since` tag here and everywhere else in this class.

Reply:
Screw `@since` :D